lib/pagseguro/transaction_request/response.rb

```diff
@@ -1,26 +1,38 @@
 module PagSeguro
   class TransactionRequest
     class Response
-      def initialize(response)
+      def initialize(response, transaction_request)
         @response = response
+        @transaction_request = transaction_request
       end
 
       def serialize
         if success?
           xml = Nokogiri::XML(response.body).css("transaction").first
-          ResponseSerializer.new(xml).serialize
+          update_attributes(ResponseSerializer.new(xml).serialize)
```

Comment:
@dmitryrck maybe it would be better to follow a standard here. every service with `update_attributes` inside the service class or everyone at response class. what do you think?